### DISCUSSION TITLE
Remember selected bandwidth on AM bands

### DISCUSSION THREAD

**Discussion by Mike86-CZE** [PE5PVB/TEF6686_ESP32#270] [Ideas]
After switching to another band it is set to 4kHz, even though I usually use 8kHz width.  It would be nice if TEF remembered the previous settings.

**Comment by PE5PVB** [comment]
Good idea. Will be implemented.

**Comment by PE5PVB** [comment]
It's now implemented. Please test.

**Comment by Mike86-CZE** [comment]
It works. Thanks
